CONTRIBUTING.md

```diff
@@ -1,16 +1,518 @@
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.4.1
+
+### Use of "Release Tracks"
+
+The Conjur Google Marketplace Application follows the recommended
+practice of releasing software in *tracks*, as described in the
+[Organizing your releases](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment#organize-releases)
+section of the Google Cloud Marketplace [Setting up your Google Cloud environment](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment) documentation.
+
+Each *track* is a series of patch semantic versions with backwards-compatible
+updates. The *track* is represented by a minor semantic version. For example,
+if the released *track* version is `1.4`, then versions `1.4.0`, `1.4.1`,
+and so on are expected to be backwards-compatible with *track* version
+`1.4`.
+
+The *track* version is what Marketplace users would use to deploy the
+Marketplace application. Basically, the *track* version that the user sees
+is mapped to the latest published patch version of the application.
+
+Publishing a new version will therefore involve tagging and pushing of:
+* Tag/push for the latest release images using patch version (e.g. `1.4.1`)
+* Tag/push for the latest release images using track version (e.g. `1.4`)
+
+### Push a *Track* Version of the Images to the GCR Registry
+
+Push a "Track" Version of the Images to the GCR Registry:
+
+```
+export NEW_VERSION=1.4.1
+export TRACK_VERSION=1.4
+
+docker tag gcr.io/conjur-cloud-launcher-onboard/cyberark:$NEW_VERSION gcr.io/conjur-cloud-launcher-onboard/cyberark:$TRACK_VERSION
+docker tag gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:$NEW_VERSION gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:$TRACK_VERSION
+docker tag gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:$NEW_VERSION gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:$TRACK_VERSION
+docker tag gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:$NEW_VERSION gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:$TRACK_VERSION
+docker tag gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:$NEW_VERSION gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:$TRACK_VERSION
+
+docker push gcr.io/conjur-cloud-launcher-onboard/cyberark:$TRACK_VERSION
+docker push gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:$TRACK_VERSION
+docker push gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:$TRACK_VERSION
+docker push gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:$TRACK_VERSION
+docker push gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:$TRACK_VERSION
+```
+
+### Install mpdev (Marketplace Development Tool)
```

Comment:
Needs a LF below

Reply:
It would be good to add a short blurb about what does `mpdev` do here without needing to click outbound links.

Reply:
Great suggestion. Added a blurb about mpdev.